test 'createMemory should have bundles' failing on Windows

This only affects people looking to contribute to the project. Probably a windows only bug. 

<details>
<summary>Edit: This section of the report is a yarn bug. Ignore.</summary>

Steps:
```
λ rm -rf node_modules\
λ yarn
yarn install v0.27.5
[1/4] Resolving packages...
[2/4] Fetching packages...
warning fsevents@1.1.2: The platform "win32" is incompatible with this module.
info "fsevents@1.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
[4/4] Building fresh packages...
$ npm run build

> autodll-webpack-plugin@0.0.6 build E:\Projects\repos\autodll-webpack-plugin
> del-cli ./lib && babel src --out-dir lib

src\compileIfNeeded.js -> lib\compileIfNeeded.js
src\createCompiler.js -> lib\createCompiler.js
src\createConfig.js -> lib\createConfig.js
src\createLogger.js -> lib\createLogger.js
src\createMemory.js -> lib\createMemory.js
src\index.js -> lib\index.js
src\normalizeEntry.js -> lib\normalizeEntry.js
src\paths.js -> lib\paths.js
src\plugin.js -> lib\plugin.js
src\utils\fs.js -> lib\utils\fs.js
src\utils\index.js -> lib\utils\index.js
Done in 20.65s.

λ yarn test
yarn test v0.27.5
$ tape -r babel-register specs/**/*.js | faucet
module.js:487
    throw err;
    ^

Error: Cannot find module 'E:\Projects\repos\autodll-webpack-plugin\-r'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at E:\Projects\repos\autodll-webpack-plugin\node_modules\faucet\node_modules\tape\bin\tape:5:5
    at Array.forEach (native)
    at Object.<anonymous> (E:\Projects\repos\autodll-webpack-plugin\node_modules\faucet\node_modules\tape\bin\tape:4:23)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
not ok 1 no plan found
not ok 2 no assertions found
⨯ fail  2
error Command failed with exit code 1.

λ npm test

> autodll-webpack-plugin@0.0.6 test E:\Projects\repos\autodll-webpack-plugin
> tape -r babel-register specs/**/*.js | faucet

module.js:487
    throw err;
    ^

Error: Cannot find module 'E:\Projects\repos\autodll-webpack-plugin\-r'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at E:\Projects\repos\autodll-webpack-plugin\node_modules\faucet\node_modules\tape\bin\tape:5:5
    at Array.forEach (native)
    at Object.<anonymous> (E:\Projects\repos\autodll-webpack-plugin\node_modules\faucet\node_modules\tape\bin\tape:4:23)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
not ok 1 no plan found
not ok 2 no assertions found
⨯ fail  2
```

This is probably not an issue with this library though.

</details>

<br>

If I do an `npm install`, the tests run but one fails:
```
λ npm test

> autodll-webpack-plugin@0.0.6 test E:\Projects\repos\autodll-webpack-plugin
> tape -r babel-register specs/**/*.js | faucet

✓ compileIfNeeded: compile: should not call getCompiler when entry is {}
✓ compileIfNeeded: should generate files
✓ compileIfNeeded: should skip when settings equals lastSettings.json
✓ createConfig
# createMemory should have bundles

E:\Projects\repos\autodll-webpack-plugin\node_modules\memory-fs\lib\MemoryFileSystem.js:44
                        throw new MemoryFileSystemError(errors.code.EINVAL, path);
                        ^
Error
    at pathToArray (E:\Projects\repos\autodll-webpack-plugin\node_modules\memory-fs\lib\MemoryFileSystem.js:44:10)
    at MemoryFileSystem.mkdirpSync (E:\Projects\repos\autodll-webpack-plugin\node_modules\memory-fs\lib\MemoryFileSystem.js:139:13)
    at createFakeFS (E:/Projects/repos/autodll-webpack-plugin/specs/createMemory.spec.js:12:6)
    at Test.t (E:/Projects/repos/autodll-webpack-plugin/specs/createMemory.spec.js:20:14)
    at Test.bound [as _cb] (E:\Projects\repos\autodll-webpack-plugin\node_modules\tape\lib\test.js:64:32)
    at Test.run (E:\Projects\repos\autodll-webpack-plugin\node_modules\tape\lib\test.js:83:10)
    at Test.bound [as run] (E:\Projects\repos\autodll-webpack-plugin\node_modules\tape\lib\test.js:64:32)
    at Immediate.next (E:\Projects\repos\autodll-webpack-plugin\node_modules\tape\lib\results.js:71:15)
    at runCallback (timers.js:800:20)
    at tryOnImmediate (timers.js:762:5)
✓ createMemory should have bundlesiateCallback] (timers.js:733:5)
not ok 7 no plan found
⨯ fail  1
```

--- a/specs/createMemory.spec.js
+++ b/specs/createMemory.spec.js
@@ -1,11 +1,12 @@
 import test from 'ava';
 import { _createMemory } from '../src/createMemory';
 import createHash from '../src/createHash';
+import os from 'os';
 import MemoryFileSystem from 'memory-fs';
 import { promisifyAll } from 'bluebird';
 import path from 'path';
 
-const cacheDir = '/my/cache/dir';
+const cacheDir = `${os.tmpdir()}/my/cache/dir`;
 
 // why? testing with the FileSystem is hard.
 // We can't run multiple tests in parallel when using the FS

